index.html

```diff
@@ -89,5 +93,147 @@ <h3 class="subtitle">I am a Software Developer</h3>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project3 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project4 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project5 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project6 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+    </section>
+    <!--about me-->
+    <section class="aboutme">
+      <h1 class="title abtitle">About me</h1>
+      <p class="paragraph">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <button class="btstyle mainbt" type="button">Get My Resume</button>
+      <img
+        class="divisory secondline"
+        src="./icons/Line2.png"
+        alt="divisory line"
+      />
```

Comment:
- [ ] This is a page decoration and not an image so it would be good to add it using CSS background images. You can learn more about background images [here](https://www.w3schools.com/cssref/pr_background-image.asp) 👍

Reply:
inserted the decorative lines using CSS as suggested, thank you!